src/Module/Profile/Profile.php

```diff
@@ -84,7 +84,7 @@ protected function rawContent(array $request = [])
 			$user = $this->database->selectFirst('user', ['uid'], ['nickname' => $this->parameters['nickname'] ?? '', 'account_removed' => false]);
 			if ($user) {
 				try {
-					$data = ActivityPub\Transmitter::getProfile($user['uid']);
+					$data = ActivityPub\Transmitter::getProfile($user['uid'], !ActivityPub::isAcceptedRequester($user['uid']));
```

Comment:
After the change to the parameter polarity.
```suggestion
					$data = ActivityPub\Transmitter::getProfile($user['uid'], ActivityPub::isAcceptedRequester($user['uid']));
```